react-d3-chart/src/components/Overlay/index.tsx

```diff
@@ -53,20 +55,23 @@ export const Overlay: React.FC<OverlayProps> = ({
     setTooltipState({ enabled: true, xOffset: mouseX, xScaled: timeX });
   };
 
+  const throttledMouseMove = _.throttle(onMouseMove, 0.1);
```

Comment:
Add a comment why this is necessary.

Something like, prevents onMouseMove from being called within 100ms of each other.